### PR TITLE
Updated add/edit monitors doc

### PR DESCRIPTION
**Pull request by akristen**
Added a direct link to the UI page to create a synthetic monitor. The UI was updated since these docs were last edited.

- Directly linked to Create monitor as a bandaid/temporary solution to the changed UI.
- Removed a step monitor video because it seemed awkward and out of place on the page. Will incorporate more videos as these docs get more thorough updates
- Added 3 screenshots from the UI. Creates a more balanced page + shows the endpoint for a successful action.
- Changed h2 Add a monitor to Create a monitor to match the verbiage in the UI. (I kept the page title add/edit, since i think its fine as a title)
- In short, wanted to make it easier for new users to Create monitor if they get a little lost in our UI.

### CODE REVIEW

**Review by barbnewrelic — APPROVED** [review]
This looks really good! Appreciate how you streamlined some of the text but at the same time made the alt text in the screenshots esp helpful.

**Comment by barbnewrelic** [comment]
@akristen  Dang it, I just noticed that you specifically requested a review from @homelessbirds  . Ignore my review!
